extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -231,8 +230,10 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
 			width: '18px'
 		}).component();
 
-		this._refreshButton.onDidClick((e) => {
-			this.loadStatus();
+		this._refreshButton.onDidClick(async (e) => {
+			this._connectionStatusLoader.loading = true;
+			await this.loadStatus();
+			this._connectionStatusLoader.loading = false;
```

Comment:
Do you want to handle the case where loadStatus might throw by putting loading = false in a finally?